src/elements/content-sidebar/activity-feed/common/activity-timestamp/_mixin.scss

```diff
@@ -0,0 +1,5 @@
+@mixin bcs-ActivityInfo {
```

Comment:
Should this mixin live at a higher level to avoid dependencies on such a specific component?